packages/xo-web/src/xo-app/index.js

```diff
@@ -230,6 +231,8 @@ export const ICON_POOL_LICENSE = {
       placeholder: '',
     },
     isXoaStatusOk: ({ xoaStatus }) => !xoaStatus.includes('✖'),
+    distinctHostVersionsByPoolId: ({ hostsByPoolId }) =>
+      mapValues(hostsByPoolId, hosts => new Set(map(hosts, 'version'))),
   },
```

Comment:
It feels a bit weird to store an unstructured collection of versions. Maybe we could simply store a boolean for each pool instead? Something like `areHostsVersionsEqual[myPool.id]`.